testdata/PSF-2.0.t1

```diff
@@ -2,44 +2,50 @@
 PSF-2.0 0,$
 
 PYTHON SOFTWARE FOUNDATION LICENSE VERSION 2
+--------------------------------------------
```

Comment:
Copied from [the current Python LICENSE](https://github.com/python/cpython/blame/ea46579067fd2d4e164d6605719ffec690c4d621/LICENSE).